organisations/README.md

```diff
@@ -0,0 +1,122 @@
+
+1. A String is added to the line array for each streetAddressLine element. The first occurrence of `addr` is mapped, 
+additional occurrences are ignored.
+2. The first occurrence of `telecom` is mapped, additional occurrences are ignored.
+3. Multiple practitioners could represent the same organisation. Therefore, the adaptor maps the first instance of 
+a represented organisation, determined by ODS code, to an `Organisation` and links the relevant 
+[Practitioners](../practioners/README.md) via `PractionerRole` resources. The `id` is appended with `"-ORG"` as the 
+[Practitioner](../practioners/README.md) mapping also uses the value mapped from `Agent / id [@root]`.
+
+If after generating the organisations above, the losing practice is not present in the Agent Directory and 
+needs to be referenced as an author for a [Document Reference](../document%20references/README.md), an Organisation 
+is generated with the losing practice ODS code.
+
+### Generated Organisation
+| Mapped to (JSON FHIR Organisation resource field) | Mapped from (XML HL7 / other)                                  |
+|---------------------------------------------------|----------------------------------------------------------------|
+| id                                                | A random UUID generated by the Adaptor                         |
+| identifier\[0].system                             | fixed value = `"https://fhir.nhs.uk/Id/ods-organization-code"` |
+| identifier\[0].value                              | The ODS code of the losing practice                            |
+
+### Unmapped fields
+
+The following Organisation fields are not currently populated by the adaptor: 
+
+- implicitRules
+- language
+- contained
+- extension
+- identifier.id
+- identifier.use
+- identifier.type
+- identifier.type.id
+- identifier.period
+- identifier.assigner
+- active
+- type.id
+- type.coding
+- alias
+- telecom.id
+- telecom.period
+- address.id
+- address.text
+- address.city
+- address.district
+- address.country
+- address.period
+- partOf
+- contact
+- endpoint
+
+## JSON FHIR > XML HL7
+
+GP Connect FHIR Organisations are mapped to the HL7v3 Agent Directory.  
+
+Where an Organisation is not a mapped as a `representedOrganisation` of an `AgentPerson` ([Practitioner](../practioners/README.md)), the adaptor maps the 
+Organisation to an `AgentPerson` instead of `AgentOrganisation`. This is due to compatability issues where attribution 
+was given to the Organisation as a performer.
+
+### Organisation as an AgentPerson 
+| Mapped to (XML HL7)         | Mapped from (JSON FHIR / other source) |
+|-----------------------------|----------------------------------------|
+| Agent / id \[@root]         | New UUID gnerated by the adaptor       |
+| Agent / code \[@nullFlavor] | fixed value = `"UNK"`                  |
+| Agent / code / originalText | fixed value = `Unknown`                |
+| Agent / name / family       | `Organisation.name`                    |
```

Comment:
Looks more like `Agent / agentPerson / name / family ` to me?

Reply:
👍